principles/fp-011-locus-of-authority.md

```diff
@@ -15,23 +15,34 @@ light of scientific advance, and for ensuring that all user feedback is addresse
 
-## Implementation
-    
-### Recommendations and Requirements
-
-The name, email address and github username of this person should be provided when requesting to register with [OBO](http://obofoundry.org). 
-This person must be subscribed to 
-obo-discuss in order to be kept abreast of community developments of relevance to 
-participating ontology projects. The email address of the person who is the locus of the 
-authority must be kept up-to-date, and before that person ceases to have responsibility 
+## Recomendations and Requirements
+A primary contact person MUST be assigned.
+The name, email address and GitHub username of the contact person MUST be provided when requesting to register with [OBO](http://obofoundry.org). The contact person MUST be subscribed to obo-discuss in order to be kept abreast of community developments of relevance to 
+participating ontology projects, but the primary contact person can, of course, delegate 
+these responsibilities for the project as necessary. The email address of the person who is the locus of the 
+authority MUST be kept up-to-date, and before that person ceases to have responsibility 
 for the project, they should identify a replacement and update the metadata accordingly 
-(via the [OBO Foundry issue tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues)) before they move on. The contact person can, of course, delegate 
-these responsibilities as necessary.
+(via the [OBO Foundry issue tracker](https://github.com/OBOFoundry/OBOFoundry.github.io/issues)) before they move on.
+
+## Implementation
+
+First, read the [FAQ](http://obofoundry.github.io/faq/how-do-i-edit-metadata.html) on how to edit the metadata for your ontology.
+
+The contact email MUST NOT be a mailing list. If this person does not already have a GitHub account, we request that they [create one](https://github.com/join). Then, add the following to your [metadata file](https://github.com/OBOFoundry/OBOFoundry.github.io/tree/master/ontology) (replacing with the correct email, name, and GitHub username):
+
+`contact:
+ email: foo@bar.com
+ label: John Smith
+ github: jsmith123`
 
 ### Examples: 
 For ChEBI, the primary contact person is Adnan Malik (amalik@ebi.ac.uk). 
 
 ### Counter-Examples: 
 Mailing list; for ChEBI, chebi-help@ebi.ac.uk
 
+### Criteria for Review
+
+Email address will be checked to ensure it is written in a standard format.
```

Comment:
Perhaps reference https://emailregex.com/

Reply:
Also maybe check that github account exists?

Reply:
@jamesaoverton can/should the github account check recommended by @cthoyt be implemented? If so, we'll indicate that it is part of the checked criteria.

Reply:
If the question about whether a technical solution could be made about github accounts, the answer is yes. You can use this API as an example:
1. An existing user https://api.github.com/users/cthoyt
2. A not existing user https://api.github.com/users/cthoyt-not-real that returns the following message:

```json
{
  "message": "Not Found",
  "documentation_url": "https://docs.github.com/rest/reference/users#get-a-user"
}
```